sagemaker_batch_transform/custom_tensorflow_inference_script_csv_and_tfrecord/custom_tensorflow_inference_script_csv_and_tfrecord.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Generally, for notebooks, we want to use the present tense instead of the future tense. This applies to other cells in the notebook as well.</p><p> </p><p>Also, remove "As such, the two datasets will be identical." The data is identical but the format is not, so it is unclear.</p><p> </p><p>Also, "Here we set up the specific TensorFlow version we will use to train and host our model." -&gt; "Here, we set up the specific TensorFlow version that we will use to train and host our model."</p><p> </p><p>Also, "IAM role arn" -&gt; "IAM role ARN"</p><p> </p><p>Also, "Please make use of an execution role with the <code>SageMakerFullAccess</code> policy attached." -&gt; "Please make sure the <code>SageMakerFullAccess</code> policy is attached to the execution role you are using."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3438/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done